<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWidget</class>
 <widget class="QWidget" name="SimulationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimulationWidget</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>319</width>
      <height>289</height>
     </rect>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>311</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="scrollLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="SaveAsButton">
        <property name="text">
         <string>Save As...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="FilePath"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="RunSimulation">
        <property name="text">
         <string>Run Simulation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="timeStepsLine"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Number of Time Steps:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>RunSimulation</sender>
   <signal>clicked()</signal>
   <receiver>SimulationWidget</receiver>
   <slot>onRunSimulationClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveAsButton</sender>
   <signal>clicked()</signal>
   <receiver>SimulationWidget</receiver>
   <slot>onSaveAsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSaveAsClicked()</slot>
  <slot>onRunSimulationClicked()</slot>
 </slots>
</ui>
